tests/bsim/bluetooth/audio/test_scripts/audio_config_tests/cap_ac_10_16_1_1.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+source $(dirname "$0")/_ac_common.sh
+
+Execute_cap_unicast_ac -c 10 -r 16_1_1 -n 1
```

Comment:
This set of per test scripts would also need to pass whatever options they got down to the `Execute_<whatever>` function (so we can pass arguments from the caller of this to the function to the phy)

Reply:
Hmm, I think this would conflict with getopts, right? 

Maybe I can use the `*)` case in `_ac_common.sh` to grab all the arguments not parsed by `_ac_common.sh` to then forward to `Execute`. Will need to try this later